test/integration/haproxy_osuosl/inspec/haproxy_osuosl_spec.rb

```diff
@@ -51,6 +51,20 @@
   end
 end
 
-describe command('ip address show eth0') do
+describe service 'keepalived' do
+  it { should be_installed }
+  it { should be_enabled }
+  it { should be_running }
+end
+
+60.times do
```

Comment:
What happens if this runs 60 times and ever completes? Do you catch this?

Reply:
The actual test is outside the loop so if it gets run even if the loop never breaks